app/bundles/DynamicContentBundle/Views/DynamicContent/list.html.twig

```diff
@@ -0,0 +1,184 @@
+                                    delete: securityHasEntityAccess(
+                                        permissions['dynamiccontent:dynamiccontents:deleteown'],
+                                        permissions['dynamiccontent:dynamiccontents:deleteother'],
+                                        item.getCreatedBy()
+                                    ),
+                                },
+                                    routeBase: 'dynamicContent',
+                                    nameGetter: 'getName',
+                                }) }}
+                        </td>
+                        <td>
+                            {{- include(
+                                'MauticCoreBundle:Helper:publishstatus_icon.html.twig',
+                                {
+                                    item: item, model: 'dynamicContent'
+                                }) }}
+
+                            <a href="{{ url(
+                                'mautic_dynamicContent_action',
+                                {objectAction: 'view', objectId: item.getId()}
+                            ) }}" data-toggle="ajax">
+                                {{ item.getName() }}
+                                {% set hasVariants     = item.isVariant() %}
+                                {% set hasTranslations = item.isTranslation() %}
+                                {% set isFilterBased   = item.getIsCampaignBased()|default(true) %}
+
+                                {% if (hasVariants or hasTranslations or isFilterBased) %}
+                                    <span>
+                                {% if (hasVariants) %}
+                                    <span data-toggle="tooltip"
+                                          title="{% trans %}mautic.core.icon_tooltip.ab_test{% endtrans %}">
+                                            <i class="fa fa-fw fa-sitemap"></i>
+                                        </span>
+                                {% endif %}
+                                        {% if (hasTranslations) %}
+                                            <span data-toggle="tooltip"
+                                                  title="{% trans %}mautic.core.icon_tooltip.translation{% endtrans %}">
+                                        <i class="fa fa-fw fa-language"></i>
+                                    </span>
+                                        {% endif %}
+                                 </span>
+                                {% endif %}
+                            </a>
+                        </td>
+                        <td class="visible-md visible-lg">{{ item.getSlotName() }}</td>
+                        <td class="visible-md visible-lg">
+                            {% set category = item.getCategory() %}
+                            {% set catName = category ? category.getTitle() : 'Emtpy category' %}
```

Comment:
`<?php $catName  = ($category) ? $category->getTitle() : $view['translator']->trans('mautic.core.form.uncategorized'); ?>`

Should use the translator to have it display "Uncategorized"

Reply:
@JoshuaEstes Just fixed it. Please review it again